docs/user-guide/longitudinal-profile.md

```diff
@@ -0,0 +1,87 @@
+# Longitudinal Profile
+
+*******************
+
+Given a DEM or a bathymetric layer as a source and a provided path on it, this tool allows to calculate the **Longitudinal Profile** and display it within an interactive chart.
+
+!!! note
+    The **Longitudinal Profile** is not active by default in the MapStore configuration but it can be configured within an [application contexts](application-context.md#configure-plugins) if needed.
```

Comment:
```suggestion
    The **Longitudinal Profile** is not included by default in the MapStore configuration but it can be configured within an [application contexts](application-context.md#configure-plugins) if needed.
```